### PR TITLE
AppCenter: User_UI_US10/11/12/13/15: Fix display of application center's left panel.

### PR DESCRIPTION
**Pull request by mdiha**

### CODE REVIEW

File: app-center-webapps/src/main/webapp/skin/less/app-center.less [hunk 1]
```diff
@@ -485,230 +485,229 @@
...
-              font-weight: bold;
-              padding-left: 10px;
-            }
-            .image {
-              width: 45px;
-              height: 45px;
+            flex-direction: column;
+            img {
+              border-radius: 5px;
+              box-shadow: 0 0 5px darkgrey;
```
Comment (rdenarie) [inline app-center-webapps/src/main/webapp/skin/less/app-center.less]:
```suggestion
              box-shadow: 0 0 5px @darkgrey;
```

Can you check ?

File: app-center-webapps/src/main/webapp/skin/less/app-center.less [hunk 2]
```diff
@@ -485,230 +485,229 @@
...
       }
-      .v-input__slot {
-        height: 36px!important;
+      .appPaginator button {
+        background: @primaryColor;
+        border: 1px solid @primaryColor;
+        color: @baseBackgroundDefault;
+        padding: 5px 10px;
+        border-radius: 5px;
+        box-shadow: 0 0 3px #ff7c4061;
```
Comment (rdenarie) [inline app-center-webapps/src/main/webapp/skin/less/app-center.less]: Is there a variable here to use ?

File: app-center-webapps/src/main/webapp/skin/less/app-center.less [hunk 3]
```diff
@@ -485,230 +485,229 @@
...
     }
-    .favoriteAppUrl {
-      max-width: 130px;
-      h5 {
-        color: #797979;
-        font-size: 15px;
-        padding-left: 10px;
-        flex: 1;
+    .favoriteApplication {
+      border-bottom: 1px solid #efefef;
```
Comment (rdenarie) [inline app-center-webapps/src/main/webapp/skin/less/app-center.less:654]: Can we use a variable ?

File: app-center-webapps/src/main/webapp/skin/less/app-center.less [hunk 4]
```diff
@@ -485,230 +485,229 @@
...
+      display: inline-flex;
+      margin-bottom: 15px;
+      width: 100%;
+      border-radius: 8px!important;
+      .favoriteAppImage {
+        .appImage {
+          box-shadow: 0 0 5px darkgrey;
```
Comment (rdenarie) [inline app-center-webapps/src/main/webapp/skin/less/app-center.less]:
```suggestion
          box-shadow: 0 0 5px @darkgrey;
```

File: app-center-webapps/src/main/webapp/skin/less/app-center.less [hunk 5]
```diff
@@ -485,230 +485,229 @@
...
-  }
-  .maxFavorite {
-    color: @greyColorLighten1Default;
-    font-style: italic;
-    font-size: 14px;
-  }
-  .reached {
-    color: #E53935;
+    .reached {
+      color: #E53935;
```
Comment (rdenarie) [inline app-center-webapps/src/main/webapp/skin/less/app-center.less:688]: Can we use a variable ?

File: app-center-webapps/src/main/webapp/vueLocales/locale_fr.json [hunk 6]
```diff
@@ -37,9 +37,8 @@
   "appCenter.adminSetupList.actions": "Actions",
   "appCenter.adminSetupList.remove": "Supprimer",
   "appCenter.adminSetupList.search": "Rechercher",
-  "appCenter.userSetup.appDirectory": "Annuaire des applications",
-  "appCenter.userSetup.authorized.access": "Accéder",
-  "appCenter.userSetup.authorized.favorite": "Favoris",
+  "appCenter.userSetup.appCenter": "Annuaire des applications",
+  "appCenter.userSetup.authorized.open": "OUVRIR",
```
Comment (rdenarie) [inline app-center-webapps/src/main/webapp/vueLocales/locale_fr.json]: Why this label is in capital letter ?
Perhaps we should change it to normal, and capitalize it by css

File: app-center-webapps/src/main/webapp/vueLocales/locale_en.json [hunk 7]
```diff
@@ -37,9 +37,8 @@
   "appCenter.adminSetupList.actions": "Actions",
   "appCenter.adminSetupList.remove": "Remove",
   "appCenter.adminSetupList.search": "Search",
-  "appCenter.userSetup.appDirectory": "Applications directory",
-  "appCenter.userSetup.authorized.access": "Access",
-  "appCenter.userSetup.authorized.favorite": "Favorite",
+  "appCenter.userSetup.appCenter": "Application center",
+  "appCenter.userSetup.authorized.open": "OPEN",
```
Comment (rdenarie) [inline app-center-webapps/src/main/webapp/vueLocales/locale_en.json]: Why this label is in capital letter ?
Perhaps we should change it to normal, and capitalize it by css

**Review by rdenarie — CHANGES_REQUESTED** [review]

File: app-center-webapps/src/main/webapp/skin/less/app-center.less (hunk 1 above)
Comment (mdiha) [reply app-center-webapps/src/main/webapp/skin/less/app-center.less]: Okay will check, it is existant

File: app-center-webapps/src/main/webapp/skin/less/app-center.less (hunk 2 above)
Comment (mdiha) [reply app-center-webapps/src/main/webapp/skin/less/app-center.less]: Couldn't find the variable, will verify for the right one.

File: app-center-webapps/src/main/webapp/skin/less/app-center.less (hunk 3 above)
Comment (mdiha) [reply app-center-webapps/src/main/webapp/skin/less/app-center.less:654]: This color is used in platform-ui but no variable for it.

File: app-center-webapps/src/main/webapp/skin/less/app-center.less (hunk 4 above)
Comment (mdiha) [reply app-center-webapps/src/main/webapp/skin/less/app-center.less]: I will check it

File: app-center-webapps/src/main/webapp/skin/less/app-center.less (hunk 5 above)
Comment (mdiha) [reply app-center-webapps/src/main/webapp/skin/less/app-center.less:688]: Used in platform-ui but no variable for this color.

File: app-center-webapps/src/main/webapp/vueLocales/locale_fr.json (hunk 6 above)
Comment (mdiha) [reply app-center-webapps/src/main/webapp/vueLocales/locale_fr.json]: Okay will fix it to uppercase with css.

File: app-center-webapps/src/main/webapp/vueLocales/locale_en.json (hunk 7 above)
Comment (mdiha) [reply app-center-webapps/src/main/webapp/vueLocales/locale_en.json]: Okay will fix it

**Review by rdenarie — APPROVED** [review]

**Pull request merged**